src/modules/countMeals.js

```diff
@@ -0,0 +1,10 @@
+const countMeals = () => {
+  const amount = document.querySelector('.item-amount');
+  const items = document.querySelectorAll('.card');
+  let cntr = 0;
+  items.forEach(() => {
+    cntr += 1;
+  });
+  amount.textContent = `(${cntr})`;
+};
```

Comment:
please you can get number of meals from meals.length instead of counting cards

Reply:
Hi @binhussen that's part of the requirement of the project, to count the items.